index.md

```diff
@@ -19,8 +18,6 @@ Documentation is available on this site for HEAD, or
 [hackage](http://hackage.haskell.org/package/distributed-process) for the current and preceding versions of
 each library.
 
-### <a href="/rss.xml"><img src="/img/feed-icon-28x28.png"></a> Recent Activity <a class="pull-right" href="http://hackage.haskell.org/platform" ><img src="http://hackage.haskell.org/platform/icons/button-64.png"></a>
```

Comment:
Ah sorry, only meant to remove the link to haskell-platform. The button image is no longer there and I couldn't find a good replacement.